### PR TITLE
fix(chore): modify year in copyright

### PR DESCRIPTION
**Pull request by skrdgraph**
## Problem
copyright version out of sync

## Solution
change to 2022

### CODE REVIEW

**Review by MichelDiz — APPROVED** [review]

File: ee/vault_ee.go [hunk 1]
```diff
@@ -1,7 +1,8 @@
+//go:build !oss
```
Comment (MichelDiz) [inline ee/vault_ee.go:1]: Is this right? not sure if this is part of the regex sed

File: graphql/e2e/custom_logic/custom_logic_test.go [hunk 2]
```diff
@@ -2708,7 +2708,7 @@ func TestCustomDQL(t *testing.T) {
 				screen_name: "minhaj"
 				followers: 10
 			  }
-			  timestamp: "2021-02-23"
+			  timestamp: "2022-02-23"
```
Comment (MichelDiz) [inline graphql/e2e/custom_logic/custom_logic_test.go:2711]: Not sure if we need to update testing data.

File: query/query0_test.go [hunk 3]
```diff
@@ -2496,7 +2496,7 @@ func TestDateTimeQuery(t *testing.T) {
 	// Test 21
 	query = `
 {
-  q(func: between(created_at, "2021-03-28T14:41:57+30:00", "2019-03-28T15:41:57+30:00"), orderdesc: created_at) {
+  q(func: between(created_at, "2022-03-28T14:41:57+30:00", "2019-03-28T15:41:57+30:00"), orderdesc: created_at) {
```
Comment (MichelDiz) [inline query/query0_test.go:2499]: Yep, this should be undone. This can potentially break tests.

File: systest/ludicrous/run_test.go [hunk 4]
```diff
@@ -1,7 +1,8 @@
+//go:build standalone
```
Comment (MichelDiz) [inline systest/ludicrous/run_test.go:1]: again, I have found a lot of these. I can tag them all if it's really necessary

File: graphql/e2e/custom_logic/custom_logic_test.go (hunk 2 above)
Comment (skrdgraph) [reply graphql/e2e/custom_logic/custom_logic_test.go:2711]: +1 ^

File: query/query0_test.go (hunk 3 above)
Comment (skrdgraph) [reply query/query0_test.go:2499]: ^ +1

File: systest/ludicrous/run_test.go (hunk 4 above)
Comment (skrdgraph) [reply systest/ludicrous/run_test.go:1]: ^ +1

File: graphql/e2e/custom_logic/custom_logic_test.go (hunk 2 above)
Comment (skrdgraph) [reply graphql/e2e/custom_logic/custom_logic_test.go:2711]: ^ +1

File: ee/vault_ee.go (hunk 1 above)
Comment (skrdgraph) [reply ee/vault_ee.go:1]: ^ +1

**Review by SashaDgraph — COMMENTED** [review]
It looks good to me now. @skrdgraph please review.

File: systest/backup/minio/backup_test.go [hunk 5]
```diff
@@ -117,7 +117,7 @@ func TestBackupMinio(t *testing.T) {
 	dirSetup(t)
 
 	// Send backup request.
-	// TODO: minio backup request fails when the environment is not ready, 
+	// TODO: minio backup request fails when the environment is not ready,
```
Comment (skrdgraph) [inline systest/backup/minio/backup_test.go:120]: @SashaDgraph I guess this was missed in your checks, but it's okay (an extra space was cleaned up ~ possibly by your IDE)

**Comment by skrdgraph** [comment]
👍 This PR looks good @SashaDgraph. I did a full review

@dshekhar95 - could you review this as well? This PR is touching 400+files, I did catch one small unrelated change - where I have left a comment. The best way to check the diff's is from your local IDE. We will wait for your review & then merge.

**Pull request merged**